items/armor/armor.json

```diff
@@ -709,6 +769,56 @@
 		"stamina_regen": 100,
 		"passive": 0
 	},
+	"1740717999": {
+		"name": "CW-4 Arctic Ranger",
+		"description": "",
+		"type": 0,
+		"slot": 2,
+		"armor_rating": 50,
+		"speed": 550,
+		"stamina_regen": 125,
+		"passive": 2
+	},
+	"2829569639": {
+		"name": "CW-4 Arctic Ranger",
+		"description": "",
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"1335058381": {
+		"name": "CW-9 White Wolf",
+		"description": "The pattern used for this armor is the result of the Winter Camouflage Improvement Effort, which considered 136 entrants before determining the color white would best blend in with snow.",
+		"type": 2,
+		"slot": 2,
+		"armor_rating": 150,
+		"speed": 500,
+		"stamina_regen": 100,
+		"passive": 1
+	},
+	"2538783678": {
+		"name": "CW-9 White Wolf",
+		"description": "The pattern used for this armor is the result of the Winter Camouflage Improvement Effort, which considered 136 entrants before determining the color white would best blend in with snow.",
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"4282790522": {
+		"name": "Dissident's Nightmare",
+		"description": "",
```

Comment:
no description on this one?

Reply:
Yee, haven't got any descriptions from crowd sourcing and don't got the medals for it myself to see